src/app/containers/HomePage/index.tsx

```diff
@@ -49,7 +79,213 @@ const Chip = styled(Box)`
   line-height: 1.4;
 `;
 
+const client = new W3CWebSocket(`ws://35.240.219.80:8000/wss`);
+
 export function HomePage() {
+  const { account } = useIconReact();
+
+  useChangeAccount()(`${account}`);
```

Comment:
Why we use `useChangeAccount`?
I think the `account` from useIconReact is enough.